messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/ActiveMQBroker.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.extension.messaging.activemq;
+
+import org.apache.activemq.artemis.api.core.RoutingType;
+import org.apache.activemq.artemis.api.core.SimpleString;
+import org.apache.activemq.artemis.api.core.TransportConfiguration;
+import org.apache.activemq.artemis.api.core.management.ActiveMQServerControl;
+import org.apache.activemq.artemis.core.security.SecurityAuth;
+
+/**
+ *
+ * @author Emmanuel Hugonnet (c) 2020 Red Hat, Inc.
+ */
+public interface ActiveMQBroker {
+
+    Object getDelegate();
```

Comment:
This one needs javadoc too.

I assume the other methods are just copied from ActiveMQServer or ManagementService, so could be doced with a simple "@see {@link .....#..."

Reply:
I wrapped also call to classes present in the artemis server jar via the ActiveMQServer